lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const { innerRadiusPercent, percent, ...svgAttrs } = props;
+      const innerRadiusPercentNotNull = innerRadiusPercent == null ? 50 : innerRadiusPercent < 0 ? 0 :
+        innerRadiusPercent > 100 ? 100 : innerRadiusPercent;
+
+      const strokeWidth = 100 - innerRadiusPercentNotNull;
+      const r = innerRadiusPercentNotNull + (strokeWidth / 2);
+
+      // Add 0.5π because we want the angle from the top of the circle, not the right
+      const arcEndAngle = (Math.PI / 2) + (2 * Math.PI * (percent / 100));
+      const arcEndX = r * Math.cos(arcEndAngle);
+      // Multiply by -1 because the y-axis origin (0,0) is at the top-left rather than bottom-left
+      const arcEndY = -1 * r * Math.sin(arcEndAngle);
+      const largeArc = percent > 50 ? 1 : 0;
+
+      return (
+        <svg ref={ref} viewBox="-100 -100 200 200" className="nx-binary-donut-chart" {...svgAttrs}>
```

Comment:
You'll want to manually merge the className here with any className that comes in on the props.  You can use the `classnames` utility for that, as demonstrated in many of the other components.  Otherwise a `className` specified by the caller will overwrite `nx-binary-donut-chart`.

Reply:
> You'll want to manually merge the className here with any className that comes in on the props. You can use the `classnames` utility for that, as demonstrated in many of the other components. Otherwise a `className` specified by the caller will overwrite `nx-binary-donut-chart`.

Done d08eba2ce5e2da5bcf001ffde1cc40e342e7e2c7

Reply:
> Done d08eba2

That is not at all what I meant.  Observe the usage of `classnames` in `NxCloseButton` as an example.

Reply:
> > Done [d08eba2](https://github.com/sonatype/sonatype-react-shared-components/commit/d08eba2ce5e2da5bcf001ffde1cc40e342e7e2c7)
> 
> That is not at all what I meant. Observe the usage of `classnames` in `NxCloseButton` as an example.

Changed 655a6b02744a48b1a1446c23e842355303057d68